src/modules/helper.js

```diff
@@ -1,15 +1,45 @@
+import countHomeItems from './counter.js';
+
 const gameID = process.env.GAMEID;
 
 // grab main tag displaying astronauts items
 const astronautList = document.getElementById('astronaut');
 
+// function to Post Like
+export const addNewLike = async (newLike) => {
+  try {
+    const req = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${gameID}/likes`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(newLike),
+    });
+
+    if (!req.ok) {
+      return req.status;
+    }
+    const prevLikeTag = document.getElementById(`${newLike.item_id}`);
+    const previosLikeCount = Number(document.getElementById(`${newLike.item_id}`).innerText);
+    prevLikeTag.innerHTML = previosLikeCount + 1;
+    return req;
+  } catch (error) {
+    return error;
+  }
+};
+
 // create arrow function to loop through and display austraunauts lists from API
 export const displayAstronauts = (astronauts) => {
   astronautList.innerHTML = '';
   astronauts.forEach((astronaut) => {
     // append div to main tag
-    astronautList.insertAdjacentHTML('beforeend', `<div class='person'><img class='person-img' src= '${astronaut.image}' alt='${astronaut.name}-image'> <div class='title-area'><h2>${astronaut.name}</h2> <i class="heart fa-regular fa-heart" id="${astronaut.id}"></i> </div> <button class="comment-${astronaut.id}">Comments</button><button>Reservations</button></div>`);
-
+    astronautList.insertAdjacentHTML('beforeend', `<div class='person'><img class='person-img' src= '${astronaut.image}' alt='${astronaut.name}-image'> <div class='title-area'><h2>${astronaut.name}</h2> <i class="heart fa-regular fa-heart" id="${astronaut.id}">0</i> </div> <button class="comment-${astronaut.id}">Comments</button><button>Reservations</button></div>`);
```

Comment:
Kindly create a variable for your HTML element instead. This will make your code more organized and clean.